library/ix-dev/community/netbootxyz/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+name: netbootxyz
+description: netboot.xyz lets you PXE boot various operating system installers or utilities from a single tool over the network.
+annotations:
+  title: netbootxyz
+type: application
+version: 1.0.1
+apiVersion: v2
+appVersion: 2.0.76
+kubeVersion: '>=1.16.0-0'
+maintainers:
+  - name: truenas
+    url: https://www.truenas.com/
+    email: dev@ixsystems.com
+dependencies:
+  - name: common
+    repository: file://../../../common
+    version: 1.2.6
+home: https://github.com/truenas/charts/tree/master/community/netbootxyz
+icon: https://netboot.xyz/img/nbxyz-logo.svg
```

Comment:
TODO: I'll sent you the links for the icons later, once I upload them to our s3 bucket!